### DISCUSSION TITLE
When logging a screenshot, use Logger.info instead of IO.puts

### DISCUSSION THREAD

**Discussion by msimonborg** [keathley/wallaby#667] [General]
I use RingLogger to keep my logs out of stdout unless I explicitly call for them. When I passed `log: true` as an option to `Wallaby.Browser.take_screenshot/2` I assumed that it would respect my logging configuration rather than ignore it. Is there a reason why this option uses `IO.puts` instead of `Logger.info`?

**Comment by mhanberg** [comment]
It already used IO.puts when I came onto to the project, so I couldn't really tell you why.

We would not use Logger.info tho, because most projects have their test env configured to only show warning and error logs, so it would be confusing to attempt to log it and not see it.

I am not familiar with RingLogger, is there a reason why you don't want anything printed to stdout when you run your tests?
